README.md

```diff
@@ -127,19 +127,19 @@ public @ResponseBody Response<Message> buyACar(@PathVariable("id") Long idPerson
 
 **Obs:** There is others models of response that you can configure. See more in the **Main settings** section.
 
-#### Roxana framework has a lot more features than what it is exposed here, so, if you are interesting check the example application [Amazing Book Store](https://github.com/rooting-company/amazing-book-store) or the documentation guide for more details.
+#### Roxana framework has a lot more features than what it is exposed here, so, if you are interested, check the example application [Amazing Book Store](https://github.com/rooting-company/amazing-book-store) or the documentation guide for more details.
 
 ## Main settings
 
-Roxana have some configurations that changes the application behavior. The configuration should be placed in the spring configuration yml or properties file.
+Roxana has some configurations that changes the application behavior. The configuration should be placed in the spring configuration yml or properties file.
 
 ### Properties:
 
-- **roxana.business.response-strategy:** Defines how the user's massage will be formated in the rest responses. There is four options that you can use:
+- **roxana.business.response-strategy:** Defines how the user's message will be formated in the rest responses. There are four options that you can use:
 
-  * **FULLY:** It Basically return the translated message and everything used to compose it, such like the language, the parameters and key. 
+  * **FULLY:** It basically returns the translated message and everything used to compose it, such as the language, the parameters and key. 
```

Comment:
acho que fica melhor sem o "the" do "the parameters".

Reply:
Vou ajustar geral: such as language, parameters and key.